main_exercise17.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+from exercise17 import calculate_male_bac, calculate_female_bac
+
+def main():
+    BAC_LEGAL_LIMIT = 0.08
+
+    gender = input("Enter your gender: male or female. ")
+    alcohol_amount = int(input("Enter the number of ounces of alcohol you've consumed: "))
+    weight = int(input("Enter your weight: "))
+    MESSAGE = print("Enter the number of hours since your last drink.")
```

Comment:
You can just call `print("Enter the number of hours since your last drink.")`. It will print to the console. No need to store it to a constant/variable. It's not being used later on.